src/to_vec.jl

```diff
@@ -107,3 +107,13 @@ function to_vec(d::Dict)
     end
     return d_vec, Dict_from_vec
 end
+
+# Enable basic conversion of generators
+# just identity for now to avoid doing the wrong thing for closures
```

Comment:
we can use `fieldcount` on the type of the function to detect if it is a closure.

Reply:
That's a good point, so we could just raise an error if the `fieldcount` is nonzero for now.